### PR TITLE
mining: Daeyalt essence timers

### LINKED ISSUE #11812: Daeyalt essence timers
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always
frustrated when [...]

Currently, there are some nice swirlies that indicate the active daeyalt essence node. They are noticeable, but are hard to notice when the screen is zoomed out far enough to be able to click any node without adjusting the screen

**Describe the solution you'd like**

A more clear indication of the active node, maybe functioning like the active node indicators in tears of guthix.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've
considered.

A simple solution like the green indicators of agility obstacles, arrows indicating the direction to run might be too extra. Currently I have each node tagged, the moving pink lines are much easier to notice than the faint snow. Another option might be indicating mine-able nodes with a pickaxe, like how motherlode mine works.

**Additional context**

image

daeyalt2

### PR DESCRIPTION
**Pull request by Celtech**
Adds time overlays to the daeyalt essence mines.

Preview  of enhancement

Closes #11812

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java [hunk 1]
```diff
@@ -128,20 +128,37 @@ public void onGameObjectDespawned(GameObjectDespawned event)
...
+		//Inverse despawn timer to track deyalt ore life
+		if (rock == Rock.DAEYALT)
+		{
+			final int region = client.getLocalPlayer().getWorldLocation().getRegionID();
+			RockRespawn rockRespawn = new RockRespawn(rock, object.getWorldLocation(), Instant.now(), (int) rock.getRespawnTime(region).toMillis(), rock.getZOffset());
+			respawns.add(rockRespawn);
+		}
 
 		// If the Lovakite ore respawns before the timer is up, remove it
 		if (Rock.getRock(object.getId()) == Rock.LOVAKITE)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java:164]: You should just reuse `rock` here.

**Comment by Adam-** [comment]
What is the reason why this has to be inverted compared to the other timers?

**Comment by Adam-** [comment]
I've tested this ingame and the rocks appear to regularly last more than `DEYALT_MAX_RESPAWN_TIME`. The wiki says 55-60 seconds, which is ~100 ticks. Can you look at the timings again?

File: runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java (hunk 1 above)
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java:164]:
```suggestion
		else if (rock == Rock.LOVAKITE)
```

**Comment by BobertSands** [comment]
Sorry to gravedig this merged PR, but is there any reason why this change did not also allow the Daeyalt mining to trigger the "MINING" and "NOT MINING" interface?

**Comment by Nightfirecat** [comment]
Unless I'm mistaken that overlay is currently only shown for rocks with more discrete mined ores, and that doesn't really fit well with daeyalt fragments being stackable.

**Pull request merged**